.travis.yml

```diff
@@ -1,10 +1,12 @@
 language: go
 
-sudo: required
+os: linux
+dist: bionic
 
 go:
   - "1.12"
```

Comment:
I'd be ok with dropping 1.12 altogether too - 1.13 introduces new error handling (and I like `%w`) and so far we committed to supporting only the last two versions. Additionally we should switch to the "1.14.x" syntax so that Travis-CI can pick the latest minor version automatically

Reply:
good idea on dropping 1.12, I'll do it in a separate PR as it needs a go.mod update as well